packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -158,9 +155,9 @@ if (DEBUG) {
   BUILTIN_HELPERS['-disallow-dynamic-resolution'] = disallowDynamicResolution;
 }
 
-const BUILTIN_KEYWORD_MODIFIERS: Record<string, ModifierDefinitionState> = {
-  action: actionModifier,
-};
+// With the implementation of RFC #1006, the `action` modifer was removed. It was the
```

Comment:
```suggestion
// With the implementation of [RFC #1006](https://rfcs.emberjs.com/id/1006-deprecate-action-template-helper), the `action` modifer was removed. It was the
```

small edit to add rfc url for later and to kick ci